docs/workloads/azure/backend/java/quickstart_java.md

```diff
@@ -20,62 +20,73 @@ keywords:
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+<!-- markdownlint-disable MD029 -->
+
 ## Using the repository to build the Java Spring Boot Rest API application
 
 ## Running the application locally
 
 1. ### Clone one of the Java workloads to your local machine from one of the following repos
     
-   1. Simple web API: [stacks-java repository](https://github.com/amido/stacks-java)
-   2. Web API with CQRS: [stacks-java-cqrs repository](https://github.com/amido/stacks-java-cqrs)
-   3. Web API with CQRS and events: [stacks-java-cqrs-events repository](https://github.com/amido/stacks-java-cqrs-events)
+    1. Simple web API: [stacks-java repository](https://github.com/amido/stacks-java)
+    2. Web API with CQRS: [stacks-java-cqrs repository](https://github.com/amido/stacks-java-cqrs)
+    3. Web API with CQRS and events: [stacks-java-cqrs-events repository](https://github.com/amido/stacks-java-cqrs-events)
    
-2. ### Configure required environment variables
-    
-    :::note
-    The application is currently configured to work with the Azure environment only.
-    :::
+2. ### Configure the application
+
+:::note
+The application is currently configured to work with the Azure environment only.
+:::
 
-    It uses an Azure **CosmosDB** database to store the example application data. So you should have access to an instance to use with the application.
+  It uses an Azure **CosmosDB** database to store the example application data. So you should have access to an instance to use with the application.
    
-    :::note
-    For running on a local environment you can use the [Cosmos DB emulator](docs/workloads/azure/backend/java/setting_up_cosmos_db_locally_java) (CosmosDB Emulator has a known fixed key). There is no need for CosmosDB for the simple web API implementation (1.i above) as there is no persistence layer in it. 
-    For further info please follow the [link](https://docs.microsoft.com/en-us/azure/cosmos-db/local-emulator?tabs=ssl-netstd21).
-    :::
+:::note
+For running on a local environment you can use the [Cosmos DB emulator](/docs/workloads/azure/backend/java/setting_up_cosmos_db_locally_java) (CosmosDB Emulator has a known fixed key). There is no need for CosmosDB for the simple web API implementation (1.i above) as there is no persistence layer in it. 
+For further info please follow the [link](https://docs.microsoft.com/en-us/azure/cosmos-db/local-emulator?tabs=ssl-netstd21).
+:::
+
+Set the cosmosdb URI, databaseName and key in main application configuration file (`application.yml`) using the values coming from the CosmosDB Emulator UI.
+
+```yaml title=application.yml
+azure:
+ cosmosdb:
+   uri: xxxxxx
+   database: xxxxxx
+   key: xxxxxx
+```
 
-    ---
+In addition, Azure **ApplicationInsights** is used for logging purposes. If this is unavailable, modify the application so that it doesn't fail to startup if it can't access ApplicationInsights, and simply log to the terminal instead.
```

Comment:
The log file in the local profile shouldn't contain any reference to ApplicationInsights not sure about the application.yml file but we could remove any references as well. Maybe it would be worthy to mention that in the documentation, instead

Reply:
I've added:
> Alternatively, you can use Spring `local` profile, which is configured to not require cloud infrastructure to run. For the details, please follow the [Spring Profiles](https://www.baeldung.com/spring-profiles#4-jvm-system-parameter) article.